client/src/Components/StaffRegistration/staffRegistration.js

```diff
@@ -0,0 +1,113 @@
+                <img src={staffRegoImg} alt="staffRegoImg" class="staffRegoImg" />
+                <form class="staffRegoForm" action="../Login/login.js">
+                    <h1 class="staffRego">Register Staff</h1>
+                    <table class="tableNames" align="center">
+                        <tr>
+                            <td><label for="firstName" class="firstNameLbl"><b>First Name</b></label></td>
+                            <td><label for="lastName" class="lastNameLbl"><b>Last Name</b></label></td>
+                        </tr>
+                        <tr>
+                            <td><input type="text" placeholder="First Name" name="firstName" class="firstName" required value={this.state.firstName} onChange={this.handleChange} /></td>
+                            <td><input type="text" placeholder="Last Name" name="lastName" class="lastName" required value={this.state.lastName} onChange={this.handleChange} /></td>
+                        </tr>
+                    </table>
+
+                    <table class="tableMobandGen" align="center">
+                        <tr>
+                            <td><label for="mobNum" class="mobNumLbl"><b>Mobile Number</b></label></td>
+                            <td><label for="gender" class="genderLbl"><b>Gender</b></label></td>
+                        </tr>
+                        <tr>
+                            <td><input type="text" placeholder="Mobile Number" name="mobNum" class="mobNum" maxLength={10} required value={this.state.mobNum} onChange={this.handleChange} /></td>
+                            <td>
+                                <select name="gender" class="gender" value={this.state.gender} onChange={this.handleChange}>
+                                    <option selected value="" disabled hidden />
+                                    <option value="female">Female</option>
+                                    <option value="male">Male</option>
+                                    <option value="other">Other</option>
+                                    <option value="none">Prefer not to say</option>
+                                </select>
+                            </td>
+                        </tr>
+                    </table>
+
+                    <label for="email" class="emailLbl"><b>Email Address</b></label>
+                    <input type="email" placeholder="Email Address" name="email" class="emailInput" required value={this.state.email} onChange={this.handleChange} />
+
+                    <label for="role" class="roleLbl"><b>Role</b></label>
+                    <select name="role" class="roleSelect" value={this.state.role} onChange={this.handleChange} >
+                        <option selected value="" disabled hidden />
+                        <option value="wait">Waiter/Waitress</option>
+                        <option value="chef">Chef</option>
+                        <option value="otherRole">Other</option>
```

Comment:
So I added role in the registration because, depending on the role what they see when they log in will be different. So like business owner will be the only one who can see the page with a staff list and then add new staff, everyone else sees a page with a list of orders instead (Hope that makes sense). so maybe leave the waiter and chef, but change the other role to business owner. And maybe if you want to add one that says store manager

Reply:
Ooooo, yeah! Good thinking! Thanks, I'll change it now 😃